lib/Dialect/XTenNN/IR/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@
 #
 # (c) Copyright 2022 Advanced Micro Devices, Inc.
 
-add_mlir_library(XTenNNIR
+add_mlir_library(XTenNNDialect
```

Comment:
Good catch. However, it is still not right. It must be `add_mlir_dialect_library` such that `${MLIR_DIALECT_LIBS}` is populated correctly in CMake.

I am unsure why this wasn't the case before, this may have been an oversight on my part.